src/public/server.js

```diff
@@ -183,10 +183,20 @@ Dungeon.prototype = {
     });
 
     this.socket.on('join-game', function (payload) {
-      var controller = find(controllers, payload.gameId);
-      var game = controller ? controller.game : undefined;
-      self.createArea(payload.areaWidth, payload.areaHeight);
-      if (game && !game.started) self.joinRoom(game);
+      var syncId = payload.gameId;
```

Comment:
why `syncId` ?
It's a notion we never have in the application, i find it rather confusing.
I think `gameId` is more appropriate and understandable.

Reply:
I called it syncId cause we use this id to get controller as well as corresponding dungeon. I understood this id as a way to associate objects and i felt this name more expressive.